.vib/external-dns/vib-publish.json

```diff
@@ -31,13 +31,22 @@
         }
       },
       "actions": [
+        {
+          "action_id": "goss",
+          "params": {
+            "resources": {
+              "path": "/.vib/external-dns/goss"
+            },
+            "remote": {
+              "workload": "deploy-external-dns"
+            }
+          }
+        },
```

Comment:
I don't really think this makes a difference in functioning, but usually `trivy` is the first action defined in the array of the verification step. 

We append new actions to the end of the existing array (I think this is the convention for the other files) so I'd move it to the end 😁.